src/Groups/matrices/linear_centralizer.jl

```diff
@@ -131,14 +132,14 @@ function _centr_unipotent(F::FinField, V::AbstractVector{Int}; isSL=false)
    for i in 1:length(L)-1
       pos += L[i][1]*L[i][2]
       # block above diagonal
-      z = idN
+      z = identity_matrix(F,n)
```

Comment:
This seems as if the code before was buggy? But no test caught it... ? That suggest to me that we need better tests for this function... Or I might be misunderstanding...

Reply:
No, the code was not buggy. I replaced the functions `insert_block!` and `insert_block(A,B)` with the pre-existing function `Hecke._copy_matrix_into_matrix(A,B)`. The difference is that the function `insert_block(A,B)` returns a matrix, but leaves the original value of `A` unchanged. This is not possible with `Hecke._copy_matrix_into_matrix`, that modifies the matrix `A`. This required a bit of code rewriting.

Reply:
Thanks for the explanation